docs/api.md

```diff
@@ -725,6 +735,40 @@ Determines whether or not boss bar creates fog
 
 Determines what color the boss bar color is, one of `pink`, `blue`, `red`, `green`, `yellow`, `purple`, `white`
 
+### mineflayer.Particle
+
+#### Particle.id
+
+Particle ID, as defined in the [protocol](https://wiki.vg/Protocol#Particle)
+
+#### Particle.name
+
+Particle Name, as defined in the [protocol](https://wiki.vg/Protocol#Particle)
+
+#### Particle.longDistance
+
+Determines whether or not to increase the particle distance
```

Comment:
Can you explain this a bit more and where it applies ? Not familiar with this concept in game.

Reply:
It's only use is for rendering purposes. In game if you use the `/particles` command and set the `force` option it will force any player to render the particle(even with particles set to minimum) and increase the distance it can be seen from 256 blocks to 65536 blocks.

Reply:
ok, you should rename it to something better (longDistanceRender) and put that explanation into the doc

Reply:
Renamed to `forceRender` and detailed in docs